<ui version="4.0">
 <author/>
 <comment/>
 <exportmacro/>
 <class>ChatServerForm</class>
 <widget class="QWidget" name="ChatServerForm">
 <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>660</width>
     <height>588</height>
    </rect>
   </property>
   <property name="windowTitle">
    <string>Chatting Server</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QVBoxLayout" name="clientListVerticalLayout">
        <item>
         <widget class="QLabel" name="clientListLabel">
          <property name="text">
           <string>Client List</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTreeWidget" name="clientTreeWidget">
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="headerHidden">
           <bool>false</bool>
          </property>
          <column>
           <property name="text">
            <string>State</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>ID</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Name</string>
           </property>
          </column>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget1">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTreeWidget" name="messageTreeWidget">
          <property name="rootIsDecorated">
           <bool>false</bool>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
          <column>
           <property name="text">
            <string>Sender IP(Port)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Sender ID(Name)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Message</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Receiver IP(Port)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Receiver ID(Name)</string>
           </property>
          </column>
          <column>
           <property name="text">
            <string>Time</string>
           </property>
          </column>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="messageLayout">
          <item>
           <widget class="QPushButton" name="clearPushButton">
            <property name="text">
             <string>Clear</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="savePushButton">
            <property name="text">
             <string>Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <resources/>
  <connections>
   <connection>
    <sender>clearPushButton</sender>
    <signal>clicked()</signal>
    <receiver>messageTreeWidget</receiver>
    <slot>clear()</slot>
    <hints>
     <hint type="sourcelabel">
      <x>270</x>
      <y>295</y>
     </hint>
     <hint type="destinationlabel">
      <x>288</x>
      <y>195</y>
     </hint>
    </hints>
   </connection>
  </connections>
 </ui>
